Library/Formula/ndjbdns.rb

```diff
@@ -0,0 +1,106 @@
+class Ndjbdns < Formula
+  desc "N-DJBDNS is a brand new release of DJBDNS."
+  homepage "http://pjp.dgplug.org/djbdns/"
+  url "http://pjp.dgplug.org/djbdns/ndjbdns-1.06.tar.gz"
+  sha256 "5ce5a7c5031f220a85fc8bca903f2d3cf484ff77e4c85e7144a0e2a5922a1127"
+  
+  depends_on "autoconf"
```

Comment:
These should either be `=> :build` or `=> :run` dependencies. It looks like the former.